bot/NanoTrasenBot.py

```diff
@@ -1056,7 +1056,6 @@ def autoRecv():
                      conn.privmsg(targetchannel,sender+" : "+C_sarcasticball.sarcasticball(influx.lower(),debug,truesender,nonhighlight_names,prefix)) 
             elif influx.lower()[0:len(Name)] == lowname and not influx.lower()[len(Name):].isalpha() or \

```

Comment:
@Iamgoofball 
2 options:

1: Remove the part of the ```elif``` after the ```or``` that mentions ```SName```, that's a list of shortnames that the but will respond to, like "Sibyl" "Basil" instead of their proper Mrs_Sibyl, Mr_Basil names.
2: Remove everything in this ```elif```, as at the moment the bot would still be registering that you've spoken to it, without responding, which is just weird.